srcpkgs/soju/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'soju'
+pkgname=soju
+version=0.2.1
+revision=1
+build_style=go
+go_import_path="git.sr.ht/~emersion/soju"
+go_package="./cmd/soju ./cmd/sojuctl"
+hostmakedepends="scdoc"
+short_desc="User-friendly IRC bouncer"
+maintainer="flupe <lucas@escot.me>"
+license="AGPL-3.0-or-later"
+homepage="https://soju.im"
+changelog="https://git.sr.ht/~emersion/soju/refs/v${version}"
+distfiles="https://git.sr.ht/~emersion/soju/refs/download/v${version}/soju-${version}.tar.gz"
+checksum=3558db9f78da3d0311b45e42307ef01af3218156b0370d85a1d7eef4d2811559
+conf_files="/etc/soju/config"
+system_accounts="_soju"
+make_dirs="/var/db/soju 0750 _soju _soju
+ /var/log/soju 0750 _soju _soju
+ /etc/soju 0755 root root"
```

Comment:
Since we ship a file in the `/etc/soju/` directory and its owned by root, adding it to `make_dirs`.